docs-js/orchestration/chat-completion.mdx

```diff
@@ -122,24 +120,28 @@ Check the [SAP Notes](https://me.sap.com/notes/3437766) for all available LLMs o
 
 ## Templating
 
-Use the orchestration client with templating to pass a prompt containing placeholders that will be replaced with input parameters during a chat completion request.
+Use the orchestration client with templating to pass a static template containing placeholders that will be replaced with input parameters during a chat completion request.
 This allows for variations in the prompt based on the input parameters.
 
 ```ts
-const templating: TemplatingModuleConfig = {
-  template: [{ role: 'user', content: 'What is the capital of {{?country}}?' }]
-};
 
 const orchestrationClient = new OrchestrationClient({
   llm,
-  templating
+  templating: {
+    template: [{ role: 'user', content: 'What is the capital of {{?country}}?' }]
+  }
 });
 
 const response = await orchestrationClient.chatCompletion({
   inputParams: { country: 'France' }
 });
 ```
 
+::: info
+The messages in `template` are sent to the LLM everytime you invoke `.chatCompletion()`.
```

Comment:
[q] I don't remember the style guide very clearly any more. But do we put a `.` in front of a method name?

Reply:
I also don't understand why `everytime` is reported by vale 😅